commands/command_fsck.go

```diff
@@ -46,6 +47,13 @@ func fsckCommand(cmd *cobra.Command, args []string) {
 		}
 	})
 
+	// If 'lfs.fetchexclude' is set and 'git lfs fsck' is run after the
+	// initial fetch (i.e., has elected to fetch a subset of Git LFS
+	// objects), the "missing" ones will fail the fsck.
+	//
+	// Attach a filepathfilter to avoid _only_ the excluded paths.
+	gitscanner.Filter = filepathfilter.New(nil, cfg.FetchExcludePaths())
```

Comment:
I wonder if we should use the `includePaths` here too? Could you think of a situation where this would be necessary? E.g. is it possible to exclude `/foo/*` and include `/foo/bar` (I have never tried that).

Reply:
> I wonder if we should use the `includePaths` here too?

I don't think so, precisely since an include filter does not override an exclude filter. In the above example, the filter composed of including `/foo/bar` and excluding `/foo/*` would _not_ match `/foo/bar`.

The reason that I only rely on the exclude filter is because we want to check _all_ files, (regardless if we're only including a subset of them), but we definitely _don't_ want to check objects that we haven't downloaded.

Does this seem OK?

Reply:
👍